api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -288,6 +288,50 @@ public void saveConcept_shouldForceSetFlagIfSetMembersExist() {
 		assertTrue(concept.getSet());
 	}
 	
+	/**
+	 * @see ConceptServiceImpl#saveDrug(Drug) 
+	 */
+	@Test
+	public void saveDrug_shouldPutGeneratedIdOntoReturnedDrug() {
+		executeDataSet("org/openmrs/api/include/ConceptServiceTest-initialConcepts.xml");
+		//New drug instance
```

Comment:
We don't need to keep these comments as any from the programming background will understand this.